src/experimental-layers/src/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -373,7 +373,7 @@ export default class SolidPolygonLayer extends Layer {
           uniforms: {
             isSideVertex: 1
           },
-          isinstanced: 1,
+          isInstanced: 1,
```

Comment:
Not for this PR: Should we change these to `true` to celebrate our new prop name?